ludwig/automl/base_config.py

```diff
@@ -41,10 +41,18 @@
 PATH_HERE = os.path.abspath(os.path.dirname(__file__))
 CONFIG_DIR = os.path.join(PATH_HERE, 'defaults')
 
-model_defaults = {
-    'concat': os.path.join(CONFIG_DIR, 'concat_config.yaml'),
-    'tabnet': os.path.join(CONFIG_DIR, 'tabnet_config.yaml'),
-    'transformer': os.path.join(CONFIG_DIR, 'transformer_config.yaml')
+BASE_AUTOML_CONFIG = os.path.join(CONFIG_DIR, "base_automl_config.yaml")
+
+combiner_defaults = {
+    "concat": os.path.join(CONFIG_DIR, "combiner/concat_config.yaml"),
+    "tabnet": os.path.join(CONFIG_DIR, "combiner/tabnet_config.yaml"),
+    "transformer": os.path.join(CONFIG_DIR, "combiner/transformer_config.yaml"),
+}
+
+encoder_defaults = {
+    "text": {
+        "bert": os.path.join(CONFIG_DIR, "text/bert_config.yaml"),
```

Comment:
While BERT gets very good performance, I feel like it's cost/performance ratio is pretty bad (running on a single v100, for example, the batch size ends up being so small you'd be better off using something else like DistilBERT). Finding a batch size small enough to work for BERT in such cases is also a challenge. But curious if you've observed something different.

Reply:
I personally don't know about this; would need to check with Avanika here.
In the meantime, I could pull this update out of the change set, while we get more clarity here,
if you think that would be best.

Reply:
Sounds good, we can try it out and see how it goes.